easybuild/easyconfigs/s/SCALCE/SCALCE-2.7-GCC-4.8.2.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'MakeCp'
+
+name = 'SCALCE'
+version = '2.7'
+
+homepage = 'http://scalce.sourceforge.net/Home'
+description = """SCALCE [skeɪlz] is a FASTQ compression tool that uses locally consistet parsing
+ to obtain better compression rate. SCALCE has been specifically designed for Illumina reads but it can
+ handle other technologies (that generate base pair reads) if the read length is the same throughout the file."""
+
+toolchain = {'name': 'GCC', 'version': '4.8.2'}
+
+# eg. http://sourceforge.net/projects/scalce/files/2.7/scalce-v2.7-src.tar.gz/download
+sources = ['%(namelower)s-v%(version)s-src.tar.gz']
+source_urls = [SOURCEFORGE_SOURCE]
+
+preconfigopts = 'cd zlib-1.2.8 &&'
```

Comment:
The resulting installation doesn't provide zlib, or require the included zlib to be around, so it's fine I guess.

However, funnily enough, it also depends on `bzip2`:

```
$ ldd /gpfs/scratch/projects/project_gpilot/vsc40023/software/SCALCE/2.7-GCC-4.8.2/bin/scalce 
        linux-vdso.so.1 =>  (0x00007fffbbbff000)
        libbz2.so.1 => /lib64/libbz2.so.1 (0x00002b3b9a8ef000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00002b3b9ab00000)
        libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00002b3b9ad1e000)
        libm.so.6 => /lib64/libm.so.6 (0x00002b3b9b024000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00002b3b9b2a8000)
        libc.so.6 => /lib64/libc.so.6 (0x00002b3b9b4bf000)
        /lib64/ld-linux-x86-64.so.2 (0x00002b3b9a6cd000)
```

so, we should include a dependency on `bzip2` in here?